### DISCUSSION TITLE
How can I make a recurring donation to ReThinkDNS?

### DISCUSSION THREAD

**Discussion by Braintoe** [celzero/rethink-app#1809] [General]
How can I make a recurring donation to ReThinkDNS? Stripe only lets me do a one-time donation whenever I remember (which I just did and made), but a smaller automatic donation every month would be... well, more convenient.

**Comment by ignoramous** [comment]
Thanks for your sponsorship! (:

> a smaller automatic donation every month would be... well, more convenient.

GitHub Sponsors (which uses Stripe) supports recurring payments: https://github.com/sponsors/serverless-dns
